index.html

```diff
@@ -0,0 +1,471 @@
+            </a>
+          </li>
+
+        </ul>
+
+        <div class="content_paragraph">
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+
+        <div class="tech_list">
+          <ul class="list">
+            <li>
+              <a href="">
+                <img src="./images/htmltag.jpg" alt="html">
+              </a>
+            </li>
+
+            <li>
+              <a href="">
+                <img src="./images/csstag.jpg" alt="css">
+              </a>
+            </li>
+
+            <li>
+              <a href="">
+                <img src="./images/jstag.jpg" alt="js">
+              </a>
+            </li>
+          </ul>
+        </div>
+
+        <a href="">
+          <img class="button" src="./images/SeeProject.jpg" alt="">
+        </a>
+      </div>
+    </section>
+
+
+    <section class="container4">
+      <div>
+        <div class="Pics">
+
+          <img src="./images/multi-posts2.jpg" alt="professional" />
+        </div>
+
+        <div class="tonic">
+          <img src="./images/multipoststoriestitle.jpg" alt="multiposts">
+        </div>
+
+        <ul class="dots">
+          <li class="canopy">
+            <a href="">
+              <img src="./images/canopytext.jpg" alt="canopy">
+            </a>
+          </li>
+
+          <li class="dot1">
+            <a href="">
+              <img src="./images/dot.jpg" alt="dot1">
+            </a>
+          </li>
+
+          <li class="backend">
+            <a href="">
+              <img src="./images/backenddev.jpg" alt="backend">
+            </a>
+          </li>
+
+          <li class="dot2">
+            <a href="">
+              <img src="./images/dot.jpg" alt="dot2">
+            </a>
+          </li>
+
+          <li class="year">
+            <a href="">
+              <img src="./images/2015.jpg" alt="year">
+            </a>
+          </li>
+
+        </ul>
+
+        <div class="content_paragraph">
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+
+        <div class="tech_list">
+          <ul class="list">
+            <li>
+              <a href="">
+                <img src="./images/htmltag.jpg" alt="html">
+              </a>
+            </li>
+
+            <li>
+              <a href="">
+                <img src="./images/csstag.jpg" alt="css">
+              </a>
+            </li>
+
+            <li>
+              <a href="">
+                <img src="./images/jstag.jpg" alt="js">
+              </a>
+            </li>
+          </ul>
+        </div>
+
+        <a href="">
+          <img class="button" src="./images/SeeProject.jpg" alt="">
+        </a>
+      </div>
+    </section>
+
+    <section class="container5">
+      <div class="Pics">
+        <img src="./images/aboutmyselftitle.jpg" alt="title">
```

Comment:
- [ ] As stated earlier, please kindly use a `hx` tag here instead of an `img` tag.